### ISSUE TITLE
Suggest standard URLSearchParams alternative in README

### ISSUE THREAD

**Issue by silverwind** [sindresorhus/query-string#307]
The URL standard includes the `URLSearchParams` interface which in many cases should be able to replace this module. I think it should be suggested as an alternative.

````js
> String(new URLSearchParams({a: 1, b: 2}))
'a=1&b=2'
> Object.fromEntries(new URLSearchParams('a=1&b=2'))
{ a: '1', b: '2' }
````

**Comment by sindresorhus** [comment]
I agree. I've been thinking about this. I do t think it's useful to just link to it though. We should explain in detail how it differs and they trade-offs.

**Comment by nerdyman** [comment]
I recently wrote an article detailing the differences between `query-string` and `URLSearchParams`. Not sure if it's in-depth enough but it might be useful?

https://dev.to/nerdyman/replacing-query-string-with-native-urlsearchparams-4kdg

**Comment by tredondo** [comment]
It's been a year. Maybe it would help to point to this issue in the README and leave it up to the developer to decide if they want to use the standard object?

**Comment by jimmywarting** [comment]
A bonus point for using URLSearchParams instead is that the fetch api and some other will automatically insert the correct content-type header when using URLSearchParams as a body.

**Comment by jasonsaayman** [comment]
Thanks I have merged the documentation update 😊

**Comment by sindresorhus** [comment]
I'm just gonna add it as a tip. I will probably never get to adding proper comparison.
